### PR TITLE
ILS: translations update

### PR DESCRIPTION
**Pull request by kkrampa**

### CODE REVIEW

File: .tx/config [hunk 1]
```diff
@@ -9,4 +9,5 @@ trans.fr = locale/fra/LC_MESSAGES/django.po
 trans.es = locale/es/LC_MESSAGES/django.po
 trans.hi = locale/hin/LC_MESSAGES/django.po
 trans.pt = locale/por/LC_MESSAGES/django.po
+trans.sw = locale/sw/LC_MESSAGES/django.po
```
Comment (kkrampa) [inline .tx/config]: @czue I'm not sure that is all that should be done to have translations in transifex. Could you ping someone responsible for transifex, please?

Comment (czue) [inline .tx/config]: @esoergel think you were looking into this but can you follow up here?

**Comment by czue** [comment]
good to merge from my side but will let @esoergel follow up

File: .tx/config (hunk 1 above)
Comment (esoergel) [inline .tx/config]: This isn't strictly necessary, as this is the default directory transifex searches for anyways, but I like being explicit about it (I actually just did the [same change](https://github.com/dimagi/commcare-hq/commit/a9261f731d4a36f83d9ca49dba9fc9e63e5ebbaf)).

The language also needs to be added through the transifex UI, but someone did that a while ago.  The only other thing that's a bit tricky is that if you edit the `.po` files locally, you also need to update the timestamp in the metadata at the top and push that to transifex or else transifex will ignore the edits, and they'll get overwritten next time someone pulls translations.  I pulled in the old translations which had been manually added from an old commit [here](https://github.com/dimagi/commcare-hq/commit/2820c5d98a165718bb1a8eb347eca9324257edde).

This PR tags the strings for translation again, so next time someone runs `makemessages`, it should take the old translations (currently stored but not used) and apply them to these strings.

**Comment by czue** [comment]
@esoergel was hard to tell from your comment what was actionable. can this be merged?

**Comment by kkrampa** [comment]
@czue @esoergel Updated

**Comment by esoergel** [comment]
Yeah, sorry, my comment was kinda rambly.  I mostly meant that yes, I think that this is sufficent - re-tagging the strings for translation will bring in the old translations next time someone runs `makemessages`.
@kkrampa did that, so I'll merge, pull, and push to transifex.

**Comment by esoergel** [comment]
Okay, I've pushed that to transifex.  It looks like all the translations in `custom/ilsgateway/tanzania/reminders` are showing up there, although there are [6 untranslated](https://www.transifex.com/dimagi/commcare-hq/translate/#sw/djangopo/23396158?translated=no&file=custom%2Filsgateway%2Ftanzania%2Freminders).
I've notified @modonnell729 and he'll take care of getting those last few translated.

**Comment by czue** [comment]
thanks!

**Pull request merged**